src/SolutionTemplate/UnoSolutionTemplate.WinUI.netcore/Server/Program.cs

```diff
@@ -20,10 +20,10 @@ public static void Main(string[] args)
 
 			app.UseAuthorization();
 
-$if$($UseWebAssembly$ == True)
+#if USE_WASM
 			app.UseUnoFrameworkFiles();
 			app.MapFallbackToFile("index.html");
-$endif$
+#endif
```

Comment:
Why is this an issue?

Reply:
It gets replaced with `<!-- -->` which is invalid.

Reply:
I see. There's a C# syntax for it, but it's an annoying one, that will do for now.